### PR TITLE
Add guide for screego

### LINKED ISSUE #1799: [screego] add guide
[screego](https://screego.net/) is an open-source software that provides screen sharing via WebRTC.

### PR DESCRIPTION
**Pull request by no-one**
Fixes #1799

### CODE REVIEW

**Comment by no-one** [comment]
A question for the person who will review this pull request later: I have not currently entered a port range for the TURN server. There is the following example in the config:
``` EditorConfig
# Limit the ports that TURN will use for data relaying.
# Format: min:max
# Example:
#   50000:55000
SCREEGO_TURN_PORT_RANGE=
```
In most cases this will probably not be so relevant (because STUN can be used instead of TURN) but if this is not the case: As you can see, a fairly large range is specified. However, ports can only be opened individually. What do you suggest?

File: source/guide_screego.rst [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+Configuration
+=============
+
+Configure Screego
+-----------------
+
+.. note:: You can use the following command to generate your own ``SCREEGO_SECRET``: ``tr -dc A-Za-z0-9 < /dev/urandom | head -c 40; echo``
+
+.. warning:: Replace ``isabell`` with your username, ``X940rRflEAWtVQvtPmCKgsj1WoJ8IoNZ2tEaDSxZ`` with your secret and ``40130``/``40131`` with your ports!
```
Comment (nichtmax) [inline source/guide_screego.rst:77]: This looks like a real secret, people will just copy-paste it. How about `YourSecret`?

Comment (nichtmax) [reply source/guide_screego.rst:77]: Check out https://lab.uberspace.de/guide_gitea/ for instance, there's a secret as well.

**Review by nichtmax — CHANGES_REQUESTED** [review]
Thanks for your guide! I left you some comments, I'll test everything when these are resolved.

File: source/guide_screego.rst (hunk 1 above)
Comment (no-one) [reply source/guide_screego.rst]: You're right, I've changed it. However, I used “snake case” as in the Gitea guide.

File: source/guide_screego.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+If it's not in state RUNNING, check your configuration.
+
+Open website
+------------
+.. warning:: Replace ``isabell`` with your username and ``40130`` with your port!
+
+If Screego is running, you can find the website here:
+
+  * https://isabell.uber.space:40130/
```
Comment (nichtmax) [inline source/guide_screego.rst:128]: Is there a reason you're not configuring [a backend](https://manual.uberspace.de/web-backends/)? Their docs say:
> If TLS should be enabled for HTTP requests. Screego requires TLS,
> you either have to enable this setting or serve TLS via a reverse proxy.

So I would just disable TLS here, you then only have to open one port and serve the webinterface via backend. This solves another problem: On a fresh asteroid there is no certificate until the first request, screego fails to start in this case.

**Review by nichtmax — CHANGES_REQUESTED** [review]
One small thing left ☺️

File: source/guide_screego.rst (hunk 2 above)
Comment (no-one) [reply source/guide_screego.rst]: You're right, I've changed it.

**Comment by nichtmax** [comment]
Thanks! Please get in touch via hallo@uberspace.de for your goodie!

**Pull request merged**